go-controller/pkg/util/gateway-init.go

```diff
@@ -40,68 +40,77 @@ func getLocalSystemID() (string, error) {
 	return localSystemID, nil
 }
 
-func lockNBForGateways() error {
-	localSystemID, err := getLocalSystemID()
+// GetDefaultGatewayRouterIP returns the first gateway logical router name
+// and IP address as listed in the OVN database
```

Comment:
Woudn't this be a problem as more gateway routers are added? We could keep getting a new default gateway router based on how "ovn-nbctl find" sorts?